crates/services/consensus_module/poa/src/service_test/trigger_tests.rs

```diff
@@ -194,6 +200,60 @@ async fn interval_trigger_produces_blocks_periodically() -> anyhow::Result<()> {
     Ok(())
 }
 
+// In case of error during block creation, the block should be retried after a delay of 1 second.
+#[tokio::test(start_paused = true)]
+async fn retry_block_creation_in_case_error() -> anyhow::Result<()> {
```

Comment:
[nit]: I don't think all the comments are necessary. 

For example, the comment above the test name could just be the test name. I don't mind verbose test names:
```rs
async fn service__if_commit_result_fails_then_retry_commit_result_after_trigger_duration
```

I think the comments above the `expect_commit_result`s are good. But a lot of the other comments are redundant.

Reply:
Ok, I removed the others comments !